src/patternfly/layouts/Stack/examples/index.js

```diff
@@ -1,23 +1,25 @@
-import React from 'react'
-import Documentation from '@siteComponents/Documentation'
-import Example from '@siteComponents/Example'
-import StackTemplate from './stack-template.hbs'
-import StackTemplateHasGutter from './stack-template-has-gutter.hbs'
-import docs from '../docs/code.md'
-import '../styles.scss'
+import React from 'react';
+import Documentation from '@siteComponents/Documentation';
+import Example from '@siteComponents/Example';
+import StackTemplateRaw from '!raw!./stack-template-example.hbs';
+import StackTemplateHasGutterRaw from '!raw!./stack-template-has-gutter-example.hbs';
+import StackTemplate from './stack-template-example.hbs';
+import StackTemplateHasGutter from './stack-template-has-gutter-example.hbs';
+import docs from '../docs/code.md';
+import '../styles.scss';
 
 export const Docs = docs;
 
 export default () => {
-  const stackTemplate = StackTemplate()
-  const stackTemplateHasGutter = StackTemplateHasGutter()
+  const stackTemplate = StackTemplate();
```

Comment:
nit picky, but we are using example and template throughout but it doesn't seem consistent. Do we want to omit them altogether or require one specifically?

Reply:
That crossed my mind, too, but all the layouts were using template so I let it be. But I agree, maybe it would be more consistent to remove `template` from them and use `example` at the end.